portfolio/src/main/webapp/index.html

```diff
@@ -299,6 +304,110 @@ <h1>Free Time</h1>
     </div>
     -->
 
+    <section id="nationalParks">
+    <div class="wrapper2" id="nationalParks">
+        <h1>National Parks</h1>
+        <hr>
+        <br>
+        <h3 style="text-align: center">Here is a map of all the national parks I have visited:</h3>
+        <br>
+        <div id="map"></div>
+        <br>
+        <br>
```

Comment:
I think you're using a lot of these `<br>` tags to add in vertical spacing between the elements. Instead, you should use padding and margin css rules; they're easier to maintain and a lot more flexible.

So this:
```
<h3>Here is a map:</h3>
<br>
<br>
<div id="map"></div>
```

Becomes:
```
<h3>Here is a map:</h3>
<div id="map"></div>
```
```
h3 {
  margin-bottom: 40px;
}
```